Simulator/include/IbInputSimulator/SendTypes/Logitech.hpp

```diff
@@ -125,13 +125,13 @@ namespace Send::Type::Internal {
                 MouseButton button;
                 Byte button_byte;
             };
-            int8_t x;
-            int8_t y;
+            int16_t x;
+            int16_t y;
```

Comment:
This struct is used by both LGS and G HUB. Changing it will break LGS and old versions of G HUB. A new send type could be added to avoid this problem, like GHUB2.

Reply:
But do we need LGS in general? They don't support mouse driver anymore, and I could not find any working version for me. It looks like drivers 9.02.61 and bellow doesn't have Win11 support and on my machine they show some driver error on start up.